.gitignore

```diff
@@ -14,4 +14,9 @@
 **/.classpath
 
 # VS Code
-.vscode
\ No newline at end of file
+.vscode
+
+#Autogenerated build files
+dependency-reduced-pom.xml
+velocity.log
+router-metrics/*.tgz
```

Comment:
Maybe this should be in router-metrics/.gitignore ?

Reply:
Thanks for feedback.
Look like need to fix Makefile and Dockerfile as router-metrics-$version.tgz created in root folder instead of /build. I will do this.